gwml2_schema.sql

```diff
@@ -0,0 +1,102 @@
+CREATE SCHEMA groundwater;
+
+CREATE TABLE groundwater.CI_Telephone (
+	id serial PRIMARY KEY,
+	voice VARCHAR(50),
+	facsimile VARCHAR(50)
+);
+
+CREATE TABLE groundwater.CI_Address (
+	id serial PRIMARY KEY,
+	deliveryPoint VARCHAR(50),
+	city VARCHAR(150),
+	administrativeArea VARCHAR(150),
+	postalCode VARCHAR(50),
+	country VARCHAR(150),
+	electronicMailAddress VARCHAR(150)
+);
+
+CREATE TABLE groundwater.CI_OnlineResource (
+	id serial PRIMARY KEY,
+	linkage VARCHAR(150),
+	protocol VARCHAR(150),
+	applicationProfile VARCHAR(250),
+	name VARCHAR(150),
+	description VARCHAR(255),
+	function groundwater.CI_OnLineFunctionTerm
+);
+
+CREATE TABLE groundwater.CI_Contact (
+	id serial PRIMARY KEY,
+	phone int4 REFERENCES groundwater.CI_Telephone(id) ON DELETE SET NULL,
+	address int4 REFERENCES groundwater.CI_Address(id) ON DELETE SET NULL,
+	onlineResource int4 REFERENCES groundwater.CI_OnlineResource(id) ON DELETE SET NULL,
+	hoursOfService VARCHAR (255),
+	contactInstructions VARCHAR (255)
+);
+
+CREATE TABLE groundwater.CI_ResponsibleParty (
+	id serial PRIMARY KEY,
+	individualName VARCHAR (255),
+	organisationName VARCHAR (255),
+	positionName VARCHAR (255),
+	contactInfo int4 REFERENCES groundwater.CI_Contact(id) ON DELETE SET NULL,
+	role groundwater.CI_RoleTerm
+);
+
+CREATE TABLE groundwater.GM_Envelope (
+	id serial PRIMARY KEY,
+	upperCorner GEOMETRY,
+	lowerCorner GEOMETRY
+);
+
+CREATE TABLE groundwater.borehole (
```

Comment:
We are missing the driller and drilling method fields here.

Make sure that when you create these term ferences you are doing m2m relations when the model indicates [0..*]

Reply:
Also check not other fields are missing

Reply:
driller is here https://github.com/kartoza/postgis-gwml2/pull/1/files#diff-045bc76ca0968b6170e442ab81dcae0bR68
drilling methods is here https://github.com/kartoza/postgis-gwml2/pull/1/files#diff-35e0e514637c09c255a0acc93b2e776dR5

i think i will combine those two files so you can just copy and paste and load them into your editor

Reply:
The table definition should be here, the table data shoudl be in vocabs

Reply:
> driller is here https://github.com/kartoza/postgis-gwml2/pull/1/files#diff-045bc76ca0968b6170e442ab81dcae0bR68
> drilling methods is here https://github.com/kartoza/postgis-gwml2/pull/1/files#diff-35e0e514637c09c255a0acc93b2e776dR5

They are attributes of the borehole entity and are not listed below:

<img width="709" alt="Screenshot 2020-06-20 at 19 31 37" src="https://user-images.githubusercontent.com/178003/85209074-f1824200-b32c-11ea-938d-5cac9c4b8ffb.png">